src/server/worker/pipeline/transform/transform_test.go

```diff
@@ -297,10 +296,10 @@ func TestTransformPipeline(suite *testing.T) {
 
 	suite.Run("TestJobSuccessEgressEmpty", func(t *testing.T) {
 		t.Parallel()
-		_, bucket := testutil.NewObjectClient(t)
+		objC := dockertestenv.NewTestObjClient(t)
 		pi := defaultPipelineInfo()
 		pi.Details.Input.Pfs.Glob = "/"
-		pi.Details.Egress = &pps.Egress{URL: fmt.Sprintf("local://%s/", bucket)}
+		pi.Details.Egress = &pps.Egress{URL: fmt.Sprintf("local://%s/", objC.BucketURL())}
```

Comment:
Wait, is this setting it to `local://minio://<host>/<bucket>`?  That can't be right

Reply:
Yeah I'm not sure why this didn't break.  Interestingly, url.Parse doesn't error with that input.
https://play.golang.org/p/CIVsAPZW2NA